docs/content/build-image.md

```diff
@@ -114,7 +114,7 @@ WORKDIR ${BUNDLE_DIR}
 CMD ["/cnab/app/run"]
 
 Starting Invocation Image Build =======>
-Step 1/9 : FROM debian:stretch
+Step 1/9 : FROM --platform=linux/amd64 debian:stretch-slim
```

Comment:
Stuff like this is why we've since moved away from including the output of some commands since they get stale very easily 😀